media-libs/libaubio/libaubio-0.4.8.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="A library to label music and sounds"
+DESCRIPTION="aubio is a library to label music and sounds. It listens to \
+audio signals and attempts to detect events. For instance, when a drum is \
+hit, at which frequency is a note, or at what tempo is a rhythmic melody.
+
+Its features include segmenting a sound file before each of its attacks, \
+performing pitch detection, tapping the beat and producing midi streams from \
+live audio."
+HOMEPAGE="https://aubio.org"
+COPYRIGHT="2003-2018 Paul Brossier"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://aubio.org/pub/aubio-$portVersion.tar.bz2"
+CHECKSUM_SHA256="dcf7473e1e123cc7f95ae06d18cf7901d70d5d245b1adaceca4c779568f84bba"
+SOURCE_DIR="aubio-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="5.4.8"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libaubio$secondaryArchSuffix = $portVersion
+	cmd:aubiomfcc$secondaryArchSuffix
+	cmd:aubionotes$secondaryArchSuffix
+	cmd:aubioonset$secondaryArchSuffix
+	cmd:aubiopitch$secondaryArchSuffix
+	cmd:aubioquiet$secondaryArchSuffix
+	cmd:aubiotrack$secondaryArchSuffix
+	lib:libaubio$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	ffmpeg$secondaryArchSuffix
```

Comment:
Does it require everything provided by ffmpeg? I think we should expand this into `lib:` entries.